test/integration/stepper/verticalStepper.spec.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-env mocha */
+import React from 'react';
+import {mount} from 'enzyme';
+import {assert} from 'chai';
+import getMuiTheme from 'src/styles/getMuiTheme';
+import VerticalLinearStepper from '../fixtures/VerticalLinearStepper';
```

Comment:
I'm wondering if we couldn't use one of the examples of the documentation for the integration test.